website/assets/js/pages/try-fleet/log-into-fleet-sandbox-and-redirect.page.js

```diff
@@ -0,0 +1,27 @@
+parasails.registerPage('log-into-fleet-sandbox-and-redirect', {
```

Comment:
Because I didn't understand the purpose of this page before, due to the API for the Fleet provisioner changing prior to my last review, I was misleading here.

Since, based on my other suggested edits in this review, the view action for this page becomes `view-sandbox-teleporter-or-redirect-because-expired`, by convention, the name of this page script, the corresponding LESS file, and the corresponding EJS file should all be `sandbox-teleporter.page.js`, `sandbox-teleporter.less`, and `sandbox-teleporter.ejs` respectively.

Make sense?